src/modules/75-cd/pages/get-started-with-cd/CDOnboardingWizardWithCLI/Steps/ViewGitopsApp.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useMemo } from 'react'
+import { useHistory, useParams } from 'react-router-dom'
+import { Button, ButtonSize, ButtonVariation, Layout } from '@harness/uicore'
+import { ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+import { useStrings } from 'framework/strings'
+import routes from '@common/RouteDefinitions'
+import { CDOnboardingSteps, WhatToDeployType, WhereAndHowToDeployType } from '../types'
+import { GITOPS_ENTITY_IDS_BY_DEPLOYMENT_TYPE } from '../Constants'
+import { useOnboardingStore } from '../Store/OnboardingStore'
+
+export default function ViewGitopsApp(): JSX.Element {
+  const { getString } = useStrings()
+  const history = useHistory()
+  const { accountId, orgIdentifier, projectIdentifier } = useParams<ProjectPathProps>()
+  const { stepsProgress } = useOnboardingStore()
+
+  const entityIds = useMemo((): { application: string; cluster: string; repo: string } => {
+    const data = stepsProgress?.[CDOnboardingSteps.WHAT_TO_DEPLOY]?.stepData as WhatToDeployType
+    const artifactId = data.artifactSubType ? data.artifactSubType?.id : (data.artifactType?.id as string)
+
+    return GITOPS_ENTITY_IDS_BY_DEPLOYMENT_TYPE[artifactId]
+  }, [])
+  const gotoGitopsApplication = (): void => {
+    const routeUrl = routes.toGitOpsApplication({
+      orgIdentifier,
+      projectIdentifier,
+      accountId,
+      module: 'cd',
+      applicationId: entityIds.application as string,
+      agentId: (stepsProgress?.[CDOnboardingSteps.WHAT_TO_DEPLOY]?.stepData as WhereAndHowToDeployType)?.agentInfo
+        ?.identifier
+    })
+
+    history.push(`${routeUrl}&tab=ResourceView`)
+  }
+  return (
```

Comment:
There's a real wall-of-code going on here that a few line breaks would really help
```suggestion
  const entityIds = useMemo((): { application: string; cluster: string; repo: string } => {
    const data = stepsProgress?.[CDOnboardingSteps.WHAT_TO_DEPLOY]?.stepData as WhatToDeployType
    const artifactId = data.artifactSubType ? data.artifactSubType?.id : (data.artifactType?.id as string)

    return GITOPS_ENTITY_IDS_BY_DEPLOYMENT_TYPE[artifactId]
  }, [])
  
  const gotoGitopsApplication = (): void => {
    const routeUrl = routes.toGitOpsApplication({
      orgIdentifier,
      projectIdentifier,
      accountId,
      module: 'cd',
      applicationId: entityIds.application as string,
      agentId: (stepsProgress?.[CDOnboardingSteps.WHAT_TO_DEPLOY]?.stepData as WhereAndHowToDeployType)?.agentInfo
        ?.identifier
    })

    history.push(`${routeUrl}&tab=ResourceView`)
  }
  
  return (
```